internal/controlplane/handlers_profile.go

```diff
@@ -422,6 +422,16 @@ func getRuleEvalStatus(
 			guidance = ruleTypeInfo.Guidance
 		}
 	}
+	remediationURL := ""
+	if dbRuleEvalStat.Entity == db.EntitiesRepository {
+		remediationURL, err = getRemediationURLFromMetadata(
+			dbRuleEvalStat.RemMetadata.RawMessage,
+			fmt.Sprintf("%s/%s", dbRuleEvalStat.RepoOwner, dbRuleEvalStat.RepoName),
+		)
+		if err != nil {
+			return nil, fmt.Errorf("parsing remediation pull request data: %w", err)
+		}
```

Comment:
Again, I suspect we'd prefer an empty URL and a log message to an entirely failed list, so I'd log the error rather than return it here.

Reply:
So the idea behind this was to avoid sending an incomplete response, in my mind it was _inconsistent_ instead of _degraded_ :)   but ok, I've changed them to log in case of an error.